### PR TITLE
[TASK] Drop DDEV configuration

### PR DESCRIPTION
**Pull request by lukaszuznanski**
Resolves: #971

To be merged after runTests.sh implementation

### CODE REVIEW

File: Documentation/Running.rst [hunk 1]
```diff
@@ -21,15 +21,10 @@ Composer scripts
...
 If you are working locally, you can run them using :bash:`composer <scriptname>`.
-If you are working with ddev, you can run them with :bash:`ddev composer <scriptname>`.
-You do not need to start or build the containers for this as this happens
-automatically.
 
 The code-quality-related Composer scripts make use of the PHIVE-installed tools.
-This means that for non-ddev-based development, you need to run :bash:`phive install`
-before you can use the Composer scripts.
 
-You can run :bash:`composer` (or :bash:`ddev composer`) to display a list of all available
```
Comment (oliverklee) [inline Documentation/Running.rst:32]: We could mention `runTests.sh composer` here.

**Comment by oliverklee** [comment]
We now have a workable `runTests.sh`. Should we resurrect this PR?

**Comment by oliverklee** [comment]
I think this deserves an entry in the changelog. What do you think?

**Comment by lukaszuznanski** [comment]
> I think this deserves an entry in the changelog. What do you think?

Definitely. Will add and resolve conflicts

**Comment by oliverklee** [comment]
Ping - let's get this in!

File: CHANGELOG.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ### Removed
 - Stop using Prophecy (#676)
+- Drop ddev configuration in favor of runTests.sh (#1063)
```
Comment (oliverklee) [inline CHANGELOG.md:40]: This needs to go into the current milestone, and it should be on top of the "Removed" section.

Also:
```suggestion
- Drop DDEV configuration in favor of `runTests.sh` (#1063)
```

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Review by oliverklee — APPROVED** [review]

**Pull request merged**